pkg/plugin/response/response.go

```
@@ -0,0 +1,181 @@
+// Data 获取data
+func (rt RespType) Data() interface{} {
+	return rt.data
+}
+
+// Result 统一响应
+func Result(c *gin.Context, resp RespType, data interface{}) {
+	if data == nil {
+		data = resp.data
+	}
+	if resp != Success {
+		c.Error(resp)
```

golangci-lint (1.64.8)

Error return value of `c.Error` is not checked

(errcheck)